src/com/ableton/VirtualEnv.groovy

```diff
@@ -0,0 +1,31 @@
+package com.ableton
+
+
+class VirtualEnv implements Serializable {
+  def script
+  def destDir
+
+  VirtualEnv(script) {
+    this.script = script
+    def pathSeparator = script.isUnix() ? "/" : "\\"
+    this.destDir = script.pwd(tmp:true) + pathSeparator + script.env.BUILD_NUMBER
+  }
+
+
+  def initialize(python) {
+    script.sh("virtualenv --python=${python} ${destDir}")
+  }
+
+
+  def installRequirements() {
+    run("pip install -r requirements.txt")
+  }
+
+
+  def run(command) {
+    this.script.sh("""
```

Comment:
I am a bit confused about when we should prefix with `this.`. Always? Never? Depends? Or should we use a specific naming convention for member variables? Such as `mFoo` or `_foo`?

Reply:
Yes, you're right, `this` isn't strictly necessary here. I'll remove it.

Regarding style... 😬 Yeah, at some point I would like to introduce a Groovy style guide, because the Java coder in me really hates the way our code looks. But if we are following standard Java conventions, there should be no prefix for fields or anything like that. It's only Google's wacky Java standards which recommend that.